kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/Constants.java

```diff
@@ -112,11 +112,16 @@ public interface Constants {
     /**
      * Global timeout in milliseconds
      */
-    long GLOBAL_TIMEOUT_MILLIS = Duration.ofMinutes(5).toMillis();
+    long GLOBAL_TIMEOUT = Duration.ofMinutes(5).toMillis();
```

Comment:
minor: the old name was better imho.   Alternative, keep them as Duration objects here, and that way there's no need to encode the unit in the variable name.

Reply:
Transformed all timeouts to `Duration` type